packages/react/src/components/DateTimePicker/__snapshots__/DateTimePicker.story.storyshot

```diff
@@ -1740,12 +1740,12 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/D
                             className="bx--number__controls"
                           >
                             <button
-                              aria-label="Decrement number"
+                              aria-label="undefined undefined"
```

Comment:
@jessieyan do we know what's going on here?

Reply:
Looks like there is a bug. We should reference the `strings` object instead of the `i18n` object

Reply:
> @jessieyan do we know what's going on here?

Looks like there is a bug. We should reference the strings object instead of the i18n object